src/factors/PartialPriorPose2.jl

```diff
@@ -18,14 +18,14 @@ function getSample(cf::CalcFactor{<:PartialPriorYawPose2})
   M = getManifold(cf.factor)
   p = identity_element(M)
   
-  Xc = rand(Z)
+  Xc = [0,0,rand(Z)]
   
   X = hat(M, p, Xc)
   points = exp(M, p, X)
   return (points, )
 end
 
-getManifold(::PartialPriorYawPose2) = SpecialOrthogonal(2)
+getManifold(::PartialPriorYawPose2) = SpecialEuclidean(2)
```

Comment:
So for partial factors, our design philosophy is to always define the `getManifold` function for that factor as the "parent" of the 'embedded' marginal/partial.

Reply:
reference added to Reqs Wiki: https://github.com/JuliaRobotics/Caesar.jl/wiki/High-Level-Requirements

Reply:
So, I currently have it that if getManifold is not defined it falls back to trying getManifoldPartial